frontend/src/App.js

```diff
@@ -207,6 +210,12 @@ class App extends Component {
         TagManager.dataLayer(tagManagerArgs);
     }
 
+    onLogoClicked = () => {
+        // A hacky way to re-mount a new gocery list by updating an arbitrary key of grocery list, 
```

Comment:
Very interesting approach! Agree that it's "hacky" in the sense that this doesn't seem to be the original intention of keys, but it might just be the cleanest way to achieve this (although not the most intuitive), so I think we are fine.